### PR TITLE
Odd fire unit

### PR DESCRIPTION
**Pull request by rusefillc**

### CODE REVIEW

File: unit_tests/engine_test_helper.h [hunk 1]
```diff
@@ -36,6 +36,16 @@ class EngineTestHelper : public EngineTestHelperBase {
 	EngineTestHelper(engine_type_e engineType, configuration_callback_t boardCallback, const std::unordered_map<SensorType, float>& sensorValues);
 	~EngineTestHelper();
 
+    // convert ms time to angle at current RPM
+    angle_t timeToAngle(float timeMs);
```
Comment (mck1117) [inline unit_tests/engine_test_helper.h:40]: stray declaration?

File: unit_tests/tests/tests.mk [hunk 2]
```diff
@@ -35,6 +35,7 @@ TESTS_SRC_CPP = \
 	tests/ignition_injection/test_fuelCut.cpp \
 	tests/ignition_injection/test_fuel_computer.cpp \
 	tests/ignition_injection/test_injector_model.cpp \
+    tests/ignition_injection/test_odd_firing_engine.cpp \
```
Comment (mck1117) [inline unit_tests/tests/tests.mk:38]: format

File: unit_tests/engine_test_helper.h (hunk 1 above)
Comment (rusefillc) [reply unit_tests/engine_test_helper.h]: fixed

File: unit_tests/tests/tests.mk (hunk 2 above)
Comment (rusefillc) [reply unit_tests/tests/tests.mk]: fixed

**Pull request merged**